src/Libraries/RevitNodesUI/RevitDropDown.cs

```diff
@@ -506,6 +529,12 @@ public Categories()
             OutPorts[0].GUID = existing.GUID;
         }
 
+        [JsonConstructor]
+        public Categories(IEnumerable<PortModel> inPorts, IEnumerable<PortModel> outPorts) : base(inPorts, outPorts)
+        {
+            // Verify additional information for output ports is not required here!
```

Comment:
should this be a TODO?

Reply:
yes, but looking at it I don't believe anything is required here